arangod/Graph/Graph.cpp

```diff
@@ -212,10 +240,31 @@ std::set<std::string> const& Graph::orphanCollections() const {
   return _orphanColls;
 }
 
+std::set<std::string> const& Graph::satelliteCollections() const {
+  return _satelliteColls;
+}
+
 std::set<std::string> const& Graph::edgeCollections() const {
   return _edgeColls;
 }
 
+bool Graph::needsToBeSatellite(std::string edge) const {
+  bool useAsSatellite = false;
+
+  auto edgeDefinition = edgeDefinitions().at(edge);
+  for (auto const& from: edgeDefinition.getFrom()) {
+    if (satelliteCollections().find(from) != satelliteCollections().end()) {
+      useAsSatellite = true;
```

Comment:
I think you can just `return true` no need to check anything else as the value will never be set to false later

Reply:
done ty, much better yes.